test/common/stats/thread_local_store_test.cc

```diff
@@ -1518,75 +1517,34 @@ TEST_F(HistogramTest, ParentHistogramBucketSummary) {
             "B3.6e+06(1,1)",
             parent_histogram->bucketSummary());
 }
-
-class ThreadLocalRealThreadsTestBase : public ThreadLocalStoreNoMocksTestBase {
+class ThreadLocalRealThreadsTestBase : public Thread::RealThreadsTestBase,
```

Comment:
nit: RealThreadsTestBase doesn't inherit from testing::Test so I wouldn't call it a base -- it's really a helper.

In fact I'm not sure you need to inherit from it: maybe you can just instantiate it as a member?

Reply:
Renamed. Technically making the helper class a member is possible, a base class is equivalent to a member initialized before all other members. But I think making it a base class sounds more intuitive to me because threading feels like an environment in which a test execute on instead of a member.